fighters/lucas/src/acmd/throws.rs

```diff
@@ -10,22 +10,39 @@ unsafe fn game_catch(fighter: &mut L2CAgentBase) {
         ArticleModule::change_motion(fighter.module_accessor, *FIGHTER_LUCAS_GENERATE_ARTICLE_HIMOHEBI, Hash40::new("catch"), false, 0.0);
         //FT_MOTION_RATE(fighter, 48.0/(69.0-1.0));
     }
+    frame(lua_state, 8.0);
+    if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 0.5);
+    }
     frame(lua_state, 9.0);
     if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 1.0);
         GrabModule::set_rebound(fighter.module_accessor, true);
+        
     }
     frame(lua_state, 12.0);
     if is_excute(fighter) {
         //CATCH(fighter, 0, Hash40::new("throw"), 3.0, 0.0, 0.0, 0.5, Some(0.0), Some(0.0), Some(-5.0), *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
-        CATCH(fighter, 0, Hash40::new("throw"), 3.0, 0.0, 0.0, 0.0, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
+        CATCH(fighter, 0, Hash40::new("top"), 3.5, 0.0, 6.3, 7.4, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
+        CATCH(fighter, 1, Hash40::new("top"), 3.5, 0.0, 6.3, 12.3, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
     }
     game_CaptureCutCommon(fighter);
+    frame(lua_state, 13.0);
+    if is_excute(fighter) {
+        CATCH(fighter, 2, Hash40::new("top"), 3.5, 0.0, 6.3, 17.0, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
+        CATCH(fighter, 3, Hash40::new("top"), 3.5, 0.0, 6.3, 20.0, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
+    }
     frame(lua_state, 14.0);
     if is_excute(fighter) {
-        //CATCH(fighter, 0, Hash40::new("throw"), 2.8, 0.0, 0.0, 0.0, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
+        grab!(fighter, *MA_MSC_CMD_GRAB_CLEAR, 0);
+        grab!(fighter, *MA_MSC_CMD_GRAB_CLEAR, 1); //Clear the first two, and set the 2 tip grabboxes to ignore aerial opponents
+        CATCH(fighter, 2, Hash40::new("top"), 3.5, 0.0, 6.3, 17.0, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_G);
+        CATCH(fighter, 3, Hash40::new("top"), 3.5, 0.0, 6.3, 20.0, None, None, None, *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_G);
+        FT_MOTION_RATE(fighter, 0.5);
```

Comment:
you could simplify this a lot with something like this (most grabs are capsules anyways):

frame 12: 
```
CATCH(fighter, 0, Hash40::new("top"), 3.5, 0.0, 6.3, 7.4, Some(0.0), Some(6.3), Some(12.3), *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
```

frame 13: 
```
CATCH(fighter, 0, Hash40::new("top"), 3.5, 0.0, 6.3, 7.4, Some(0.0), Some(6.3), Some(20.0), *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_GA);
```

frame 14: 
```
CATCH(fighter, 0, Hash40::new("top"), 3.5, 0.0, 6.3, 17.0, Some(0.0), Some(6.3), Some(20.0), *FIGHTER_STATUS_KIND_CAPTURE_PULLED, *COLLISION_SITUATION_MASK_G);
```

Reply:
basic idea: instead of writing 4 different grab boxes and disabling 0-1 after a few frames, just stretch one single grab box to fit the same desired bounds, and redefine those bounds each frame. 

If you agree, you can apply this same idea to the other grabs as well

Reply:
implemented, looks clean to me!
the only real reason i chose spheres was so it could interpolate but meh its prolly fine like this now

https://user-images.githubusercontent.com/19886440/230767510-b330ac96-5661-4093-aafe-5342a46dc8e7.mp4